staff/mario-rubio/challenges/tic-tac-toe-app/main.js

```diff
@@ -0,0 +1,13 @@
+
+
+function TurnO() {
+    var PlayerO = document.write("Tu turno, Jugador 1!!!") 
```

Comment:
Don't forget the semicolons, mate ;)

Reply:
According to convention, function names should be written in camelCase!

Reply:
Ok.  Continue working. thank you!